docs/cookbook.rst

```diff
@@ -19,3 +32,37 @@ wherever you need to do them::
     from screenpy_adapter_allure import AllureAdapter
 
     the_narrator.attach_adapter(AllureAdapter())
+
+After that the steps are attached to the allure report
+
+.. image:: ./allure_with_plugin.png
+  :width: 768
+  :alt: Screenshot without plugin has not the steps
+
+Attach Screenshot from selenium
+===============================
+
+If you are using allure in a selenium project using `ScreenpPy Selenium <https://screenpy-selenium-docs.readthedocs.io/en/latest/>`__
+
+You can attach a new screenshot to the report using the action `SaveScreenshot <https://screenpy-selenium-docs.readthedocs.io/en/latest/extended_api/actions.html#screenpy_selenium.actions.SaveScreenshot>`__
```

Comment:
We have `intersphinx` for exactly this case! First, you'll need to edit `docs/conf.py` a little—find the `intersphinx_mapping` dict and change it to this:
```python

intersphinx_mapping = {
    "screenpy": ("https://screenpy-docs.readthedocs.io/en/latest/", None),
    "screenpy_selenium": ("https://screenpy-selenium-docs.readthedocs.io/en/latest/", None),
}
```

Then, you can make this replacement here:
```rst
`SaveScreenshot <https://screenpy-selenium-docs.readthedocs.io/en/latest/extended_api/actions.html#screenpy_selenium.actions.SaveScreenshot>`__
.. to
:external+screenpy_selenium:class:`~screenpy_selenium.actions.SaveScreenshot`
```

The advantage here is that this link will still work if the URL for `SaveScreenshot` changes for some reason.